src/bvals/cc/bvals_cc_in_one.cpp

```diff
@@ -421,18 +556,23 @@ void SendAndNotify(MeshData<Real> *md) {
 //          Guarantees that buffers for MeshBlocks on the same rank are done, but MPI
 //          communication between ranks may still be in process.
 
-// TODO(pgrete) should probaly be moved to the bvals or interface folders
+// TODO(pgrete) should probably be moved to the bvals or interface folders
 TaskStatus SendBoundaryBuffers(std::shared_ptr<MeshData<Real>> &md) {
   Kokkos::Profiling::pushRegion("Task_SendBoundaryBuffers_MeshData");
 
-  auto boundary_info = md->GetSendBuffers();
-  bool cache_is_valid = boundary_info.is_allocated();
+  for (int b = 0; b < md->NumBlocks(); ++b) {
+    md->GetBlockData(b)->SetLocalNeighborAllcoated();
+  }
```

Comment:
Good point, we can definitely guard it with an "if sparse enabled". It may be enough to only set it when we reset the send buffers (which happens after blocks have moved or allocation status has changed). I'll try that and check that it doesn't break anything.